drivers/dai/intel/ssp/dai-params-intel.h

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __DAI_PARAMS_INTEL_H__
+#define __DAI_PARAMS_INTEL_H__
+
+#include <stdint.h>
+
+ /* ssc1: TINTE */
+#define SOF_DAI_INTEL_SSP_QUIRK_TINTE		(1 << 0)
```

Comment:
```DAI_INTEL_IPC3_...``` . Similar prefix changes need to be made here in this file too.

Reply:
ok